packages/core/content-releases/server/src/services/release.ts

```diff
@@ -0,0 +1,34 @@
+import { setCreatorFields } from '@strapi/utils';
+import type { LoadedStrapi } from '@strapi/types';
+import { RELEASE_MODEL_UID } from '../constants';
+import type { ReleaseData, UserInfo } from '../types';
+
+const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
+  async create(releaseData: ReleaseData, { user }: { user: UserInfo }) {
+    const releaseWithCreatorFields = await setCreatorFields({ user })(releaseData);
+
+    const release = await strapi.entityService.create(RELEASE_MODEL_UID, {
+      data: releaseWithCreatorFields,
+    });
+
+    return release;
+  },
+  async findMany(query: Record<string, unknown>) {
+    const { results, pagination } = await strapi.entityService.findPage(RELEASE_MODEL_UID, {
+      ...query,
+      populate: {
+        actions: {
+          // @ts-expect-error TS error on populate, is not considering count
+          count: true,
```

Comment:
@Marc-Roig Maybe you can help mi with this, I have a TS error here when passing count 🤔 But it's working completely fine, so maybe the type definition is wrong?

I tried first to use findWithRelationCountsPage but it's deprecated